core/src/main/java/com/graphhopper/reader/ReaderElement.java

```diff
@@ -95,6 +97,32 @@ public String getTag(String name) {
         return val;
     }
 
+    public <T> T getTagStartsWith(String keyPrefix, T defaultValue) {
```

Comment:
this one does not really get a tag that starts with some prefix (as the name suggests), but rather the tag associated with some key that starts with the given prefix. anyway, since this method is no longer needed, let's remove it ?

Reply:
ok, I remove it.